### PR TITLE
Enable more tests for Vertical Pod Autoscaler Operator

### PR DESCRIPTION
**Pull request by joelsmith**

### CODE REVIEW

File: ci-operator/jobs/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -44,3 +237,46 @@ presubmits:
...
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-vertical-pod-autoscaler-operator-master.yaml
+              name: ci-operator-master-configs
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
```
Comment (rphillips) [inline ci-operator/jobs/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master-presubmits.yaml]: Is name supposed to be empty here?

Comment (joelsmith) [reply ci-operator/jobs/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master-presubmits.yaml]: That's the way it generates from the config and I see the same thing in jobs for lots of other projects. I think it should be fine to have a blank container name for the jobs.

Comment (rphillips) [reply ci-operator/jobs/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master-presubmits.yaml]: Resolved via slack.

**Pull request merged**